src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
I am using this space for comments on "src/parquet/util/bpacking.h". Some of the unpack*_32 methods can be put in a loop. Putting them in a loop allows us to use vectorization pragmas as well.